website/docs/docs/partnerships.mdx

```diff
@@ -224,3 +224,5 @@ Terraform Enterprise integrations are workflow integrations that are validated a
 -> Contact us at [technologypartners@hashicorp.com](mailto:technologypartners@hashicorp.com) with any questions or feedback.
 
 After successfully building a Run Tasks integration with Terraform Cloud, partners should contact their Alliance Manager to validate the integration against Terraform Enterprise. Many customers deploy Terraform Enterprise in a private cloud or air-gapped environment, so partners must ensure that their product can communicate bi-directionally with Terraform Enterprise inside these types of networks.
+
+Upon successfully building Run Tasks integration with Terraform Cloud partners can reach out to their Alliance Manager to validate it against Terraform Enterprise. It’s common for customers to deploy Terraform Enterprise in a private cloud or in an air-gapped environment in which case partners need ensure that their product can communicate bi-directionally with TFE inside the customer’s network.
```

Comment:
```suggestion
Upon successfully building Run Tasks integration with Terraform Cloud partners can contact their Alliance Manager to validate it against Terraform Enterprise. Customers commonly deploy Terraform Enterprise in a private cloud or in an air-gapped environment, in which case partners need tp ensure that their product can communicate bi-directionally with TFE inside the customer’s network.
```

Suggest changing "reach out to" to "contact" to use fewer words. Suggest changing "It's common for customers to" to "Customers commonly" to use fewer words and avoid contractions.

Reply:
I think there's a misspelling here tp -> to, I'll take this suggestion in a separate commit